src/test/java/org/opensearch/security/SecurityAdminIEndpointsTests.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.security;
+
+import org.apache.http.HttpStatus;
+import org.junit.Assert;
+import org.junit.Test;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.security.test.SingleClusterTest;
+import org.opensearch.security.test.helper.file.FileHelper;
+import org.opensearch.security.test.helper.rest.RestHelper;
+
+public class SecurityAdminIEndpointsTests extends SingleClusterTest {
+
+    @Test public void testNoSSL() throws Exception {
```

Comment:
Please put @Test on a separate line.

Reply:
Good point!

Reply:
I've created a PR that enables enforcement but disables all rules, https://github.com/opensearch-project/security/compare/main...peternied:enforce-checkstyle?expand=1, if we turned these rules on we would run into 35,974 issues to resolve 😮 

### `./gradlew checkstyleMain`
```
  ...
  Checkstyle files with violations: 266
  Checkstyle violations by severity: [error:18588]
```

### `./gradlew checkstyleTest`
```
  ...
  Checkstyle files with violations: 181
  Checkstyle violations by severity: [error:17386]
```

We can enable rules on a piecemeal basis for specific rules or files

Reply:
Created https://github.com/opensearch-project/security/issues/1659